WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 108.xcdatamodel</string>
+	<string>WordPress 109.xcdatamodel</string>
```

Comment:
Since a default value for `unseenCount` is provided in WPKit, it'd be good to uncheck `optional` in the data model for it.

<img width="264" alt="Screen Shot 2020-12-16 at 5 01 36 PM" src="https://user-images.githubusercontent.com/1816888/102421020-a174f700-3fc0-11eb-99cc-6046406b5c4b.png">